vignettes/continuous-integration.Rmd

```diff
@@ -33,11 +33,11 @@ usethis::use_github_action("lint")
 This will create a workflow file called `lint.yaml` and place it in the correct location, namely in the `.github/workflows` directory of your repository.
 This file configures all the steps required to run `lintr::lint_package()` on your package.
 
-[lintr-bot](https://github.com/lintr-bot) will then add comments to the commit or pull request with the lints found and they will also be printed as
+Comments to the commit or pull request will be printed as
 [annotations](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-status-checks#types-of-status-checks-on-github)
 along side the status check on GitHub.
-If you want to disable the commenting you can set the environment variable `LINTR_COMMENT_BOT=false`.
-Additionally, if you want the builds to produce an error instead of just a warning, you can set the environment variable `LINTR_ERROR_ON_LINT=true`.
+If you want the builds to produce an error instead of just a warning, you can set the environment variable `LINTR_ERROR_ON_LINT=true`.
+This is set by default in [r-lib/actions](https://github.com/r-lib/actions/)'s `lint.yaml`.
```

Comment:
Maybe we should also mention [`lint-changed-files.yaml`](https://github.com/r-lib/actions/blob/v2-branch/examples/lint-changed-files.yaml) here?

Reply:
Makes sense IMO, I just added it.